src/api/entities/Checkpoint.ts

```diff
@@ -119,6 +121,7 @@ export class Checkpoint extends Entity<UniqueIdentifiers, HumanReadable> {
 
     const rawTicker = stringToTicker(ticker, context);
 
+    // Get one page of current Asset balances
```

Comment:
I am assuming paginated asset.balanceOf returns 0 for people who have sold all of their tokens but were included in a checkpoint.

Otherwise I think people who have had a balance in a checkpoint, but currently don't hold any now would be skipped.

Reply:
Yeah, you could potentially get several pages full of zeroes